$OpenBSD: patch-apps_gnome-mplayer_preferences_gschema_xml,v 1.9 2013/05/24 08:31:37 dcoppa Exp $

Set some sane defaults:
use xv as default video output
disable bar animation in full screen
use mplayer's software volume control
set audio device default volume to -1
remember the software volume value
set location of mplayer binary
use /dev/rcd0c as default optical device
use sndio as default audio device
use English as default language
disable nautilus plugin

--- apps.gnome-mplayer.preferences.gschema.xml.orig	Wed Jan  4 21:12:33 2012
+++ apps.gnome-mplayer.preferences.gschema.xml	Fri May 24 08:20:05 2013
@@ -1,7 +1,7 @@
 <schemalist>
   <schema id="apps.gnome-mplayer.preferences" path="/apps/gnome-mplayer/preferences/">
     <key name="vo" type="s">
-      <default>''</default>
+      <default>'xv'</default>
       <summary>Default video output</summary>
       <description>Default video output</description>
     </key>
@@ -116,7 +116,7 @@
       <description>When set to TRUE the ability to pause on mouse click is disabled</description>
     </key>
     <key name="disable-animation" type="b">
-      <default>false</default>
+      <default>true</default>
       <summary>Disable Animation</summary>
       <description>Disable animation of fullscreen control bar sliding away</description>
     </key>
@@ -136,7 +136,7 @@
       <description>When set to TRUE the cache value is always used, even for streaming media</description>
     </key>
     <key name="softvol" type="b">
-      <default>false</default>
+      <default>true</default>
       <summary>Have mplayer use the softvol option</summary>
       <description>When set to TRUE the softvol option is passed to mplayer, useful for machines without pulseaudio</description>
     </key>
@@ -191,7 +191,7 @@
       <description>Remember location on screen</description>
     </key>
     <key name="volume" type="i">
-      <default>100</default>
+      <default>-1</default>
       <summary>Default Volume</summary>
       <description>Default volume to use for mplayer, values of 0 or 100 allow usage of setting from ALSA</description>
     </key>
@@ -271,7 +271,7 @@
       <description>Configuration value for mplayer to control the number of audio channels</description>
     </key>
     <key name="remember-softvol" type="b">
-      <default>false</default>
+      <default>true</default>
       <summary>Remember the software volume</summary>
       <description>Remember the software volume value</description>
     </key>
@@ -306,17 +306,17 @@
       <description>Subtitle loading</description>
     </key>
     <key name="mplayer-bin" type="s">
-      <default>''</default>
+      <default>'${LOCALBASE}/bin/mplayer'</default>
       <summary>Location of mplayer binary if not in path</summary>
       <description>Location of mplayer binary if not in path</description>
     </key>
     <key name="mplayer-dvd-device" type="s">
-      <default>''</default>
+      <default>'/dev/rcd0c'</default>
       <summary>DVD device name</summary>
       <description>DVD device name</description>
     </key>
     <key name="audio-device-name" type="s">
-      <default>''</default>
+      <default>'SNDIO'</default>
       <summary>Name of the preferred audio device</summary>
       <description>Name of the preferred audio device</description>
     </key>
@@ -341,17 +341,17 @@
       <description>Default Key Bindings</description>
     </key>
     <key name="audio-lang" type="s">
-      <default>''</default>
+      <default>'English,eng,en'</default>
       <summary>Preferred Audio Language</summary>
       <description>Preferred Audio Language</description>
     </key>
     <key name="subtitle-lang" type="s">
-      <default>''</default>
+      <default>'English,eng,en'</default>
       <summary>Preferred Subtitle Language</summary>
       <description>Preferred Subtitle Language</description>
     </key>
     <key name="enable-nautilus-plugin" type="b">
-      <default>true</default>
+      <default>false</default>
       <summary>Enable Audio/Video plugin in Nautilus</summary>
       <description>Enable Audio/Video plugin in Nautilus</description>
     </key>    
